bylaws.md

```diff
@@ -103,10 +103,11 @@ The Fraternity of Lambda Alpha Nu exists to cultivate a strong community around
 	   The Treasurer’s primary responsibility shall be to present a written statement of accounts and transactions to the board on a semesterly basis.
 	   In the case of financial difficulty, the Treasurer shall be responsible for notifying the Board immediately and assisting in the design and execution of recovery efforts.
 5. Terms of Office:
-    1. Elected Officer posistions shall be elected on towards the end of the spring semester on a yearly basis.
+    1. Elected Officer positions shall be elected upon on the 2nd to last scheduled meeting of the academic year.
 	   In addition, an officer who is unable to complete the second semester of their duties shall be replaced in an election at the end of the fall semester.
-    2. A Member may serve in an  elected Officer position for only one full term.
-	   Members may be nominated for the same position repeatedly, unless there is compelling organizational reason to do so such re-appointments shall be frowned upon.
+    2. A Member may serve in an elected Officer position for only one full term.
+	   Members may be nominated for the same position repeatedly unless there is a compelling organizational reason to do so.
```

Comment:
The meaning of this line is unclear. I read it as:

first half: "Members may be nominated for the same position repeatedly"
second half: "members may be nominated for the same position if there is a compelling organizational reason to do so"
linked by: "unless" implying a contrast between the halves"

Is the intent: "Members may be nominated and elected to a position they have previously been elected to if there is a compelling organizational reason to do so"?

You may wish to nail down how exactly we want to treat repeat elected candidates who served a partial term (for instance: Steven Diaz's tenure as VP Service after the special election, then again the next academic year, if I remember things correctly)